MoltBrowser patch for components/permissions_strings.grdp
Upstream base: chromium/src@d331de35ca8ab1358c7a17e4bd41627d55eaef51 (156.0.8078.74)
Generated from src/components/permissions_strings.grdp by tools/chromium/overlay_to_patches.py.
src/ is still the source of truth: edit the file there and
re-run the generator (see docs/REBASING.md).

diff --git a/components/permissions_strings.grdp b/components/permissions_strings.grdp
index a9c5c50333e3b8b8adbca8b2104e636cbf41679a..9c0057855807af7a25d5068d850cd7f8d06c5340 100644
--- a/components/permissions_strings.grdp
+++ b/components/permissions_strings.grdp
@@ -55,13 +55,13 @@
       Notifications are automatically blocked for all sites except ones you allow
     </message>
     <message name="IDS_NOTIFICATION_QUIET_PERMISSION_INFOBAR_PREDICTION_SERVICE_MESSAGE" desc="Text used as notification message when requesting permission for Web Notifications via a notification. Used when the prediction service has decided that the user will be very unlikely to grant this permission.">
-      Chrome thinks you don't want notifications from this site. So, requests are hidden to reduce interruptions.
+      MoltBrowser thinks you don't want notifications from this site. So, requests are hidden to reduce interruptions.
     </message>
     <message name="IDS_LOCATION_QUIET_PERMISSION_MESSAGE_UI" desc="Text used as notification message when requesting Gelocation permission. Used when quiet UI is enabled in settings.">
       Location requests are automatically blocked for all sites except ones you allow
     </message>
     <message name="IDS_LOCATION_QUIET_PERMISSION_MESSAGE_UI_PREDICTION_SERVICE" desc="Text used as notification message when requesting permission for Geolocation. Used when the prediction service has decided that the user will be very unlikely to grant this permission.">
-      Chrome thinks you don't want to use your location on this site. So, requests are hidden to reduce interruptions.
+      MoltBrowser thinks you don't want to use your location on this site. So, requests are hidden to reduce interruptions.
     </message>
     <message name="IDS_MEDIA_CAPTURE_AUDIO_ONLY_INFOBAR_TEXT" desc="Text requesting permission for a site to access the computer's microphone.">
       <ph name="HOST">$1<ex>html5rocks.com</ex></ph> wants to use your microphone
@@ -586,10 +586,10 @@ This will otherwise be blocked by your privacy settings. This will allow the con
     You previously didn't allow <ph name="permission">$1<ex>camera</ex></ph> for this site
   </message>
   <message name ="IDS_PERMISSION_OFF_FOR_CHROME" desc="Text informing the user that they have turned the permission off for chrome">
-    To use your <ph name="permission">$1<ex>camera</ex></ph> on this site, give <ph name="short_product_name">$2<ex>Chrome</ex></ph> access
+    To use your <ph name="permission">$1<ex>camera</ex></ph> on this site, give <ph name="short_product_name">$2<ex>MoltBrowser</ex></ph> access
   </message>
   <message name ="IDS_PERMISSION_CHROME_NEEDS_PERMISSION" desc="Text informing the user in the background that Chrome needs a permission in order to use it on the site the user is visiting">
-    <ph name="short_product_name">$1<ex>Chrome</ex></ph> needs access to your device's <ph name="permission">$2<ex>camera</ex></ph>
+    <ph name="short_product_name">$1<ex>MoltBrowser</ex></ph> needs access to your device's <ph name="permission">$2<ex>camera</ex></ph>
   </message>
     <message name="IDS_GEOLOCATION_NAME_FRAGMENT" desc="Text fragment identifying the geolocation permission">
     location
